iroha-cli/interactive/impl/interactive_query_cli.cpp

```diff
@@ -119,13 +118,17 @@ namespace iroha_cli {
       local_time_ = iroha::time::now();
 
       while (is_parsing) {
-        line = promtString("> ");
+        auto line = promtString("> ");
+        if (not line.has_value()){
+          is_parsing = false;
+          break;
```

Comment:
Usually shells don't exit on empty input (e.g bash) so maybe replace it with `continue`? (look also below)